src/minecraft_old.json

```diff
@@ -0,0 +1,29 @@
+{
+    "reportArea": {
+        "start": {
+            "x": 136,
+            "y": -59,
+            "z": -65
+        },
+        "end": {
+            "x": 152,
+            "y": -51,
+            "z": -56
+        }
+    },
+    "reportBlockNames": [
+        "emerald_block",
+        "beacon",
+        "obsidian",
+        "spruce_sign",
+        "pink_stained_glass",
+        "light_blue_stained_glass",
+        "white_stained_glass"
+    ],
+    "friends": [
+        "DarkMetalMouse",
+        "ElektronMC"
+    ],
+    "sendPublicMCMessages": true,
+    "playerAssosiationRadius": 7
+}
```

Comment:
Why is this here?

Reply:
This file contains the coordinates on the real server and the other one contains test coordinates